File: Task/Sort-an-integer-array/Ruby/sort-an-integer-array.rb
nums = [2,4,3,1,2]
sorted = nums.sort      # returns a new sorted array.  'nums' is unchanged
p sorted                #=> [1, 2, 2, 3, 4]
p nums                  #=> [2, 4, 3, 1, 2]

nums.sort!              # sort 'nums' "in-place"
p nums                  #=> [1, 2, 2, 3, 4]

File: Task/Sort-an-integer-array/Zen-C/sort-an-integer-array.c
import "std/sort.zc"

fn main() {
    let a = [7, 2, 4, 9, 1, 3, 8, 6, 5];
    sort_int((int*)a, 9);
    for i in a { print "{i} " }
    println "";
}
